styles.css

```diff
@@ -23,7 +23,7 @@
   width: 200px;
   height: 74px;
   height: 28px;
-  font-family: 'Inter',sans-serif;
+  font-family: 'Inter', sans-serif;
```

Comment:
- [ ] Great work with the styling, unfortunately,your design doesn't match the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988)

**Your design**
<img width="325" alt="Screenshot 2022-09-22 at 20 42 28" src="https://user-images.githubusercontent.com/31113416/191815938-9d063423-0458-43e8-822c-10976c5e0137.png">

**Figma design**
<img width="377" alt="Screenshot 2022-09-22 at 20 45 46" src="https://user-images.githubusercontent.com/31113416/191816161-6c00b824-4136-4a38-83de-b38881f7a06f.png">

- [ ] Kindly update your components alignments to fit the Figma design.